android-o/SafeBrowsing/SafeBrowsing/MainActivity.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Android.App;
+using Android.OS;
+using Android.Webkit;
+using System;
+
+namespace SafeBrowsing
+{
+    [Activity(Label = "SafeBrowsing", MainLauncher = true, Icon = "@drawable/icon")]
+    public class MainActivity : Activity
+    {
+        public WebView webView;
+        public bool mSafeBrowsingIsInitialized;
```

Comment:
I'd prefer to have this be a public property:
`public bool IsSafeBrowsingEnabled { get; set; }`